conda_smithy/feedstock_io.py

```diff
@@ -75,7 +75,12 @@ def remove_file(filename):
 
 
 def copy_file(src, dst):
-    shutil.copy2(src, dst)
+    with io.open(src, "r", encoding="utf-8") as fh_src:
+        with io.open(dst, "w", encoding="utf-8", newline="\n") as fh_dst:
+            for line in fh_src:
+                fh_dst.write(line)
+
+    shutil.copymode(src, dst)
```

Comment:
Seems to be needed as demonstrated by the [comment]( https://github.com/conda-forge/conda-smithy/pull/407#issuecomment-270883493 ) below. Otherwise files that have been copied end up with the wrong EOL formatting.